crates/bevy_ui/src/update.rs

```diff
@@ -37,35 +37,46 @@ fn update_clipping(
     entity: Entity,
     clip: Option<Rect>,
```

Comment:
This parameter could be renamed to make clear that the clipping `Rect` is inherited from the node's parent:

```suggestion
   maybe_inherited_clip: Option<Rect>,
```